.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build_py3:
     docker:
-      - image: pyne/pyne_ubuntu_17.04:latest
+      - image: pshriwise/pshriwise:ubuntu_17.04_pymoab_pyne
```

Comment:
Don't forget to change this back once this is ready for PR to pyne/pyne

Reply:
If the python3 updates are going to be in a later PR (I don't know what the status of that is). This should be able to go back to the original pyne image.